lib/amazonka-core/src/Amazonka/Error.hs

```diff
@@ -135,22 +137,23 @@ parseXMLError ::
   [Header] ->
   ByteStringLazy ->
   Error
-parseXMLError a s h bs = decodeError a s h bs (decodeXML bs >>= go)
+parseXMLError a s h bs = decodeError a s h bs (go <$> decodeXML bs)
   where
     go x =
       serviceError a s h
-        <$> code x
-        <*> may' (firstElement "Message" x)
-        <*> may' (firstElement "RequestId" x <|> firstElement "RequestID" x)
+      (code x)
+      (may' (firstElement "Message" x))
+      (may' (firstElement "RequestId" x) <|> may' (firstElement "RequestID" x))
```

Comment:
If you re-run `ormolu` over this file, does this indent remain? I suspect it shifts across.

Reply:
It does indeed, as shown in [e71414a](https://github.com/brendanhay/amazonka/pull/779/commits/e71414a8d267e8e40d3056096c0457fce8866d1f)